pkg/console/operator/operator.go

```diff
@@ -47,7 +50,8 @@ import (
 )
 
 const (
-	controllerName = "Console"
+	controllerName           = "Console"
+	workloadFailingCondition = "WorkloadFailing"
```

Comment:
Interesting. I thought there was a set of `Available`, `Progressing`, `Failing`, only. I didn't realize there is also `PrereqsSatisfied` & `Upgradeable`, and that we can add custom.

Reply:
you can customize the failing one's ... the conditiona type just needs to be suffexed with `Failing`

Reply:
Nice